planet/cli/data.py

```diff
@@ -53,6 +53,10 @@ def parse_filter(ctx, param, value: str) -> dict:
             raise click.BadParameter('Filter does not contain valid json.',
                                      ctx=ctx,
                                      param=param)
+        if json_value == {}:
+            raise click.BadParameter('Filter does not contain valid json.',
```

Comment:
`'{}'` is valid JSON, though... how about 'Filter is empty'?

That said, I honestly don't know if empty filters are a problem. If someone searches with an empty filter, should they expect nothing or everything? I feel like the word "filter" implies the latter. Filters reduce information.

Reply:
Agreed!! I'll get rid of the empty json bringing up an error.

Reply:
Actually, looks like we need something in the filter. If not, we get this error: `Error: {"field": {"filter.type": [{"message": "'type' is a required property"}]}, "general": []}`

Reply:
Thanks for looking that up!